addons/common/functions/fnc_assignedItemFix.sqf

```diff
@@ -52,13 +63,14 @@ GVAR(AssignedItemsInfo) = [];
                 _shownItems set [4, false];
             };
         };
-    } forEach _assignedItems;
+        nil
+    } count _assignedItems;
 
     //systemChat str _shownItems;
-
-    showMap     (_shownItems select 0);
-    showCompass (_shownItems select 1);
-    showWatch   (_shownItems select 2);
-    showRadio   (_shownItems select 3);
-    showGPS     (_shownItems select 4 || {cameraOn == getConnectedUAV _unit});  //If player is activly controling a UAV, showGPS controls showing the map (m key)
+    _shownItems params ["_showMap", "_showCompass", "_showWatch", "_showRadio", "_showGPS"];
+    showMap _showMap;
+    showCompass _showCompass;
+    showWatch _showWatch;
+    showRadio _showRadio;
+    showGPS (_showGPS || {cameraOn == getConnectedUAV _unit});  //If player is activly controling a UAV, showGPS controls showing the map (m key)
 }] call FUNC(addEventHandler);
```

Comment:
This is not a function by definition. Calling this multiple times in a mission does nothing, except costing performance.